tests/integration/flow/yaml/index.yml

```diff
@@ -0,0 +1,61 @@
+!CompoundIndexer
+components:
+  - !NumpyIndexer
+    with:
+      index_filename: vec.gz
+      metric: cosine
+    metas:
+      name: vecidx
+      workspace: $JINA_TOPK_DIR
+  - !BinaryPbIndexer
+    with:
+      index_filename: doc.gz
+    metas:
+      name: docidx
+      workspace: $JINA_TOPK_DIR
+metas:
+  name: chunk_indexer
+  workspace: $JINA_TOPK_DIR
+requests:
+  on:
+    UpdateRequest:
+      - !VectorIndexDriver
```

Comment:
Are we sure that `VectorIndexDriver` will be able to cover every method? We asusme there will not be any need to add `UpdateDriver` or `DeleteDriver`?

Reply:
I had to make some change to the VectorIndexDriver but than it worked